src/components/error-message/index.md.njk

```diff
@@ -43,6 +43,12 @@ For each error:
 - use a red border to visually connect the message and the question it belongs to
 - if the error relates to specific text fields within the question, give these a red border as well
 
+To help users of some assistive technologies, like screen readers, the error message component includes a hidden 'Error:' prefix before the message.
```

Comment:
I know we talked about it previously, but is there anyone apart from users of screen readers that this will help? If not, I wonder if we just say "To help users of screen readers". If it's more complicated than that, leave as is.